pkg/storage/azureblob/cataloger.go

```diff
@@ -0,0 +1,60 @@
+package azureblob
+
+import (
+	"context"
+	"fmt"
+	"github.com/Azure/azure-storage-blob-go/azblob"
+	"github.com/gomods/athens/pkg/config"
+	"github.com/gomods/athens/pkg/errors"
+	"github.com/gomods/athens/pkg/observ"
+	"github.com/gomods/athens/pkg/paths"
+	"strings"
```

Comment:
Can you format the imports? I think go-import can do that. std lib first, empty line and other pkgs.

Reply:
Thanks @marpio . Fixed it.